Gems/Terrain/Code/Source/Components/TerrainWorldRendererComponent.cpp

```diff
@@ -85,15 +99,65 @@ namespace Terrain
                         ->Attribute(AZ::Edit::Attributes::ReadOnly, &MeshConfiguration::IsClodDisabled)
                     ;
 
+                editContext->Class<ClipmapConfiguration>("Clipmap", "Settings related to clipmap rendering")
+                    ->DataElement(AZ::Edit::UIHandlers::ComboBox, &ClipmapConfiguration::m_clipmapSize, "Clipmap image size", "The size of the clipmap image in each layer.")
+                        ->EnumAttribute(ClipmapConfiguration::ClipmapSize1024, "1024")
+                        ->EnumAttribute(ClipmapConfiguration::ClipmapSize512, "512")
+                    ->DataElement(AZ::Edit::UIHandlers::Slider, &ClipmapConfiguration::m_macroClipmapMaxRenderRadius, "Macro clipmap max render radius", "Max render radius that the lowest resolution clipmap can cover.")
+                        ->Attribute(AZ::Edit::Attributes::Step, 1.0f)
+                        ->Attribute(AZ::Edit::Attributes::Min, 1000.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMin, 2000.0f)
+                        ->Attribute(AZ::Edit::Attributes::Max, 5000.0f)
+                        ->Attribute(AZ::Edit::Attributes::SoftMax, 4000.0f)
```

Comment:
The hard min and maxes should be pretty lenient, really they're to keep things from absolutely breaking. I'd make the min and max much lower / higher. As for the soft min, I think you could get away with making theses a bit more lenient too.